ext/rbs_extension/parser.c

```diff
@@ -1836,14 +1839,18 @@ VALUE parse_mixin_member(parserstate *state, bool from_interface, position comme
   rbs_loc_add_required_child(loc, rb_intern("keyword"), keyword_range);
   rbs_loc_add_optional_child(loc, rb_intern("args"), args_range);
 
-  return rbs_ast_members_mixin(
-    klass,
-    name,
-    args,
-    annotations,
-    location,
-    get_comment(state, comment_pos.line)
-  );
+  VALUE comment = get_comment(state, comment_pos.line);
+  switch (type)
```

Comment:
Is this just an inlining of `rbs_ast_members_mixin()`?

Reply:
Yep, since we generate the specialized methods